src/main/java/com/github/pereiratostain/Main.java

```diff
@@ -1,8 +1,84 @@
 package com.github.pereiratostain;
 
+import com.github.forax.umldoc.core.Entity;
+import com.github.forax.umldoc.core.Modifier;
+import com.github.pereiratostain.generator.MermaidSchemaGenerator;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.lang.module.ModuleFinder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.Opcodes;
+
+/**
+ * Main class.
+ */
 public class Main {
 
-    public static void main(String[] args) {
+  /**
+   * Entry point of the program.
+   *
+   * @param args The arguments from the command line.
+   */
+  public static void main(String[] args) {
+    try {
+      var entities = asm();
+      try (var writer = new OutputStreamWriter(System.out, StandardCharsets.UTF_8)) {
+        var generator = new MermaidSchemaGenerator();
+        generator.generate(writer, entities);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  /**
+   * Converts a string to a Modifier.
+   *
+   * @param string The string to convert
+   * @return A value of the enum Modifier or null if
+   *          the string can't be converted
+   */
+  public static Modifier modifier(String string) {
+    return switch (string) {
+      case "+" -> Modifier.PUBLIC;
+      case "-" -> Modifier.PRIVATE;
+      case "#" -> Modifier.PACKAGE;
+      default -> null;
```

Comment:
throw an exception here, returning null is a bad idea, the user code will fail with a NPE and it will be harder to debug.